qa/suites/upgrade/quincy-x/parallel/workload/rados_api.yaml

```diff
@@ -9,4 +9,6 @@ workload:
         clients:
           client.0:
             - cls
+        env:
+          CLS_RBD_GTEST_FILTER: '-TestClsRbd.group_snap_set:TestClsRbd.group_snap_remove'
```

Comment:
How about before each entry add a line pointing to the tracker?
i.e.:
`# See https://tracker.ceph.com/issues/67759`

Should we also add a comment or something to indicate that this test should be added back once the test binaries are compatible again? And perhaps we can open another tracker to track that?

Reply:
> How about before each entry add a line pointing to the tracker? i.e.: `# See https://tracker.ceph.com/issues/67759`

The tracker and other details can be looked up with `git blame`.

> 
> Should we also add a comment or something to indicate that this test should be added back once the test binaries are compatible again? And perhaps we can open another tracker to track that?

There is no plan to make the test binaries compatible.  It wouldn't be hard to do in this particular case, but in principle these tests aren't meant to be run across versions like that, so I think it's better to just skip the ones that happen to be too strict to pass.